src/pages/PageNotFoundView.jsx

```diff
@@ -9,13 +12,20 @@ export default function PageNotFoundView() {
    ************************************/
 
   return (
-    <div className="page-not-found-view-container">
-      <div className="page-not-found-view-body">
-        <h1>404</h1>
-        <code>{location.pathname}</code>
-        <h3>Page Not Found</h3>
-        <p>Sorry, the page you are looking for doesn't seem to exist!</p>
-        <Link to="/">Go to Home</Link>
+    <div className='page-not-found-view-container'>
+      <div className='page-not-found-view-body'>
+        <div className='page-not-found-view-image-container'>
+          <NotFoundLogo />
+        </div>
+        <div className='page-not-found-view-text-container'>
+          <p>
+            404 Page not found, <strong>{location.pathname}</strong>
+          </p>
+          <p>Let's hurry; there is nothing to fear here</p>
+          <Button>
+            <Link to='/'>Take me home</Link>
```

Comment:
The button should be wrapped in a link so we don't have to click the text to be linked back home. 
```
          <Link to='/'>
            <Button>
              <span>Take me home</span>
            </Button>
          </Link>
```